cloudos/clos.py

```diff
@@ -511,14 +528,24 @@ def is_module(self, workflow_name, workspace_id, verify=True):
         """
         my_workflows_r = self.get_workflow_list(workspace_id, verify=verify)
         my_workflows = self.process_workflow_list(my_workflows_r)
-        is_module = my_workflows.loc[
+        group = my_workflows.loc[
             (my_workflows['name'] == workflow_name) & (my_workflows['archived.status'] == False),
-            'isModule']
-        if len(is_module) == 0:
+            'group']
+        if len(group) == 0:
             raise ValueError(f'No workflow found with name: {workflow_name}')
-        if len(is_module) > 1:
+        if len(group) > 1:
             raise ValueError(f'More than one workflow found with name: {workflow_name}')
-        return is_module.values[0]
+        module_groups = ['system-tools',
+                         'data-factory-data-connection-etl',
```

Comment:
Also `intermediate` workflows as well! These are workflows only used as part of orchestrator jobs i.e. jobs that call other jobs - used in some Drug Discovery and System Tool workflows